sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,620 @@
+    SUM(notification_alert_setting_not_supported) AS notification_alert_setting_not_supported,
+    SUM(notification_alert_setting_disabled) AS notification_alert_setting_disabled,
+    SUM(notification_alert_setting_enabled) AS notification_alert_setting_enabled
+  FROM
+    product_features
+  WHERE
+    submission_date >= start_date
+  GROUP BY
+    submission_date
+)
+SELECT
+  submission_date,
+  dau,
+    /*Logins*/
+  logins_autofill_failed,
+  logins_autofill_failed_users,
+  logins_autofilled,
+  logins_autofilled_users,
+  logins_management_add_tapped,
+  logins_management_add_tapped_users,
+  logins_management_logins_tapped,
+  logins_management_logins_tapped_users,
+    /*Credit Card*/
+  cc_autofill_failed,
+  cc_autofill_failed_users,
+  cc_autofill_settings_tapped,
+  cc_autofill_settings_tapped_users,
+  cc_autofill_toggle,
+  cc_autofill_toggle_users,
+  cc_autofilled,
+  cc_autofilled_users,
+  cc_form_detected,
+  cc_form_detected_users,
+  cc_save_prompt_create,
+  cc_save_prompt_create_users,
+  cc_sync_toggle,
+  cc_sync_toggle_users,
+    /*History*/
+  history_delete_tap,
+  history_delete_tap_users,
+  history_opened,
+  history_opened_users,
+  history_removed,
+  history_removed_users,
+  history_removed_all,
+  history_removed_all_users,
+  history_removed_today,
+  history_removed_today_users,
+  history_removed_today_and_yesterday,
+  history_removed_today_and_yesterday_users,
+  history_search_tap,
+  history_search_tap_users,
+    /*FxA*/
+  fxa_disconnect,
+  fxa_disconnect_users,
+  fxa_login_completed_view,
+  fxa_login_completed_view_users,
+  fxa_login_token_view,
+  fxa_login_token_view_users,
+  fxa_login_view,
+  fxa_login_view_users,
+  fxa_paired,
+  fxa_paired_users,
+  fxa_registration_code_view,
+  fxa_registration_code_view_users,
+  fxa_registration_completed_view,
+  fxa_registration_completed_view_users,
+  fxa_registration_view,
+  fxa_registration_view_users,
+  fxa_use_email,
+  fxa_use_email_users,
+    /*Privacy*/
+  private_browsing_button_tapped,
+  private_browsing_button_tapped_users,
+  private_browsing_button_tapped_enter_private_mode,
+  private_browsing_button_tapped_enter_private_mode_users,
+  private_browsing_icon_tapped,
+  private_browsing_icon_tapped_users,
+  app_icon_new_private_tab_tapped,
+  app_icon_new_private_tab_tapped_users,
+  tabs_tray_new_private_tab_tapped,
+  tabs_tray_new_private_tab_tapped_users,
+    /*Awesomebar Location*/
+  drag_location_bar,
+  drag_location_bar_users,
+  location_top,
+  location_top_users,
+  location_bottom,
+  location_bottom_users,
+    /*Notification*/
+  notification_status_authorized,
+  notification_status_authorized_users,
+  notification_status_notDetermined,
+  notification_status_denied,
+  notification_alert_setting_not_supported,
+  notification_alert_setting_disabled,
+  notification_alert_setting_enabled
+FROM
+  dau_segments
+JOIN
+  product_features_agg
+USING
+  (submission_date)
+
+
```

Comment:
I believe there should be only 1 empty line after the final line of code, more or less might cause the formatting check to fail.